SU2_CFD/include/fluid/CFluidScalar.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include "CFluidModel.hpp"
+
+class CFluidScalar : public CFluidModel {
```

Comment:
Thank you Pedro for all your comments, really appreciate the help! Question regarding the CFluidScalar class which I made final, is it better to use override or final for its members, e.g.:

void SetLaminarViscosityModel(const CConfig* config) override;
vs
void SetLaminarViscosityModel(const CConfig* config) final;

If a member function is declared without virtual and with final, it means that it already overrides a virtual function in base class, so I figure it might be preferred to use final ? Also, do I need to state final  after every function member in CFluidScalar?